ChronoStore/include/event.h

```diff
@@ -0,0 +1,11 @@
+#include<iostream>
+#ifndef EVENT_H
+#define EVENT_H
+
+
+typedef struct Event {
+    __uint64_t timeStamp;
+    char data[100];
+} Event;
```

Comment:
Can you make the size of data field a variable, maybe a macro?

Reply:
Done